### PR TITLE
refactor: create Trigger.vue in composition API

### LINKED ISSUE #57: Refactoring: trigger functions
refactoring trigger functions
* trigger api

### PR DESCRIPTION
**Pull request by better1719**
This fix #57 
refactoring trigger functions 
trigger api

### CODE REVIEW

**Comment by yezhidaizhu** [comment]
黄色警告貌似有点多呀，可以解决一下不

**Comment by better1719** [comment]
ok

File: front-end/src/views/function/components/trigger.vue [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    },
+    watch: {
+      currentFunc() {
+        this.TriggerForm.setFieldsValue({
+          functionName: this.currentFunction.name
+        })
+      }
+    },
+    methods: {
+      onClose () {
```
Comment (RobertIndie) [inline front-end/src/views/function/components/trigger.vue:92]: Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/FunctionStream/function-stream/pullRequest?prid=7834758)
```suggestion
      onClose() {
```

**Comment by RobertIndie** [comment]
Could you help resolve conflicts? @better1719

**Review by Akhil-Yuan — APPROVED** [review]
LGTM, thanks!

**Review by RobertIndie — APPROVED** [review]
Well done! Thank you.

**Pull request merged**